### DISCUSSION TITLE
bigfix/11.0/platform/Platform/Config/c_azure_entra

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#98] [Q&A]
# bigfix/11.0/platform/Platform/Config/c_azure_entra

You can use Microsoft Entra to handle authentication on BigFix.

https://help.hcl-software.com/bigfix/11.0/platform/Platform/Config/c_azure_entra.html

**Comment by JoeBrunsTR** [comment]
Does this Bigfix entry exist in the fedramp version of the Azure tenant galllery and if not, does that mean it's not fips compliant?
